operator/src/main/java/oracle/kubernetes/operator/calls/AsyncRequestStep.java

```diff
@@ -123,6 +126,90 @@ public void listenTimeoutDoubled() {
     timeoutSeconds *= 2;
   }
 
+  class AsyncRequestStepProcessing {
+
+    final Packet packet;
+    final RetryStrategy retryStrategy;
+    final String cont;
+    final AtomicBoolean didResume = new AtomicBoolean(false);
+    final ApiClient client;
+
+    public AsyncRequestStepProcessing(Packet packet, RetryStrategy retry, String cont) {
```

Comment:
JavaDoc.  What is the "cont" parameter?

Reply:
It's the same one the K8s APIs use, apparently having to do with continuing requests. Do you have a recommendation on a better name? Should I put the underscore before it, as some k8s code seems to?

Reply:
I'm okay with "cont" if I know what it is.  This is a cursor so that you can iterate over a long list.  Say, you had a namespace with 10,000 domains.  You could do a list and say give me the first 200, and then use the cont parameter to ask for the next 200 and so on. I can't understand why it's on this method though.

Reply:
It's needed for this call:

`
    // Create a call to Kubernetes that we can cancel if it doesn't succeed in time.
    private CancellableCall createCall(AsyncFiber fiber) throws ApiException {
      return factory.generate(requestParams, client, cont, new ApiCallbackImpl(this, fiber));
    }
`

That was in the original version of the class, and I don't see an easy way to get rid of it.

Reply:
Okay, resolving this issue.